hnginternship5/ziki
# edited the dropdown arrow in the sidebar

#### What does this PR do?
* It moves the dropdown arrow in the sidebar closer to the My Posts text

#### Description of Task to be completed?
* Reduce the space besides the dropdown arrow on the sidebar

#### How should this be manually tested?
* Open the site on your browser
* Type localhost:8000
* Check the sidebar to see that the arrow is closer

#### What pages did you work on?
* Page sidebar.html

#### Screenshots (mandatory)
![Screenshot_2019-05-03 Welcome to Ziki](https://user-images.githubusercontent.com/33119163/57159785-b5b1b380-6dde-11e9-87a1-46afc6b84292.png)
